tests/baselines/reference/booleanLiteralTypes1.types

```diff
@@ -1,32 +1,32 @@
 === tests/cases/conformance/types/literal/booleanLiteralTypes1.ts ===
 type A1 = true | false;
->A1 : boolean

```

Comment:
This behaviour is intended. I don't know that it's super important. It's probably even less important that

```ts
type A = { a }
type B = { b }
type T = A | B
type U = A | B
declare var t: T
declare var u: U
```

should also print either `T` or `U` (or `A | B`) for both `t` and `u`.

It's more important that:

```ts
declare var x: true | false
```

prints 'boolean' for the type of `x`